portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path

```diff
@@ -86,7 +86,7 @@
 </tr>
 <tr>
     <td>SELECT_BUTTON</td>
-    <td>//div[@class='criteria-statement']//button[@type='button']</td>
+    <td>//div[@class='criterion'] [${indexField}] //div[@class='input-group-item input-group-item-shrink input-group-append']</td>
```

Comment:
hey @ViniLopes20 , when calling `SELECT_BUTTON` you didn't specify the `{indexField}`, is it correct?

Also, try to avoid composed classes (multiple classes with empty space between), this way your selector will be very fragile.

Reply:
Yes, @alexcandidos. `{indexField}` will be specified when using the macro for testing. It will be passed as a parameter